src/config/__tests__/config.test.js

```diff
@@ -50,17 +50,17 @@ describe('config', () => {
       });
 
       it('should map config with process.env values if set', () => {
-        process.env.GP2GP_MESSENGER_REPOSITORY_ASID = 'deductionsAsid';
-        process.env.GP2GP_MESSENGER_REPOSITORY_ODS_CODE = 'deductionsOdsCode';
+        process.env.GP2GP_MESSENGER_REPOSITORY_ASID = 'repoAsid';
+        process.env.GP2GP_MESSENGER_REPOSITORY_ODS_CODE = 'REPO_ODS_CODE';
         process.env.PDS_ASID = 'pdsAsid';
         process.env.GP2GP_MESSENGER_MHS_OUTBOUND_URL = 'mhsOutboundUrl';
         process.env.NHS_ENVIRONMENT = 'nhsEnvironment';
         process.env.SERVICE_URL = 'url';
 
         expect(initializeConfig()).toEqual(
           expect.objectContaining({
-            deductionsAsid: 'deductionsAsid',
-            deductionsOdsCode: 'DEDUCTIONSODSCODE',
+            repoAsid: 'repoAsid',
+            repoOdsCode: 'REPO_ODS_CODE',
```

Comment:
Prefer lower case value

Reply:
Sorry, for this one, using lower case seems to break the test, as in the main code there is some logic to enforce ODS code to be uppercase....
So I would keep the expected value as upper case here as well.